mission_control/navigator_launch/nodes/network_keepalive.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+import rospy
+from std_msgs.msg import String
+
+
+class KeepAlive(object):
```

Comment:
For the KeepAlive system, there is a publisher and a listener. Why not make that explicit with the naming of these classes and their respective files? Maybe KeepAlivePublisher and KeepAliveListener? ... KeepAliveServer and KeepAliveClient?